frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/install.tsx

```diff
@@ -36,27 +41,41 @@ export const CreateAttachedDevicesCluster: React.FC<CreateAttachedDevicesCluster
     K8sResourceKind[]
   >(LSOSubscriptionResource);
 
+  const csvResource = {
+    kind: referenceForModel(ClusterServiceVersionModel),
+    fieldSelector: `metadata.namespace=${getLSONamespace(
```

Comment:
use getNamespace from shared folder.
```suggestion
    fieldSelector: `metadata.namespace=${getNamespace(
```